z_calibration.py

```diff
@@ -209,6 +211,12 @@ def _probe(self, mcu_endstop, z_position, speed):
             # retract
             self._move([None, None, curpos[2] + self.retract_dist],
                        self.lift_speed)
+            if wiggle:
```

Comment:
I think Klipper will do this. But I would like to have a check that at least one of the two offsets is not 0.0 too.